components/websiteMetadata.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+const { formatMessage } = useIntl()
+const title = ref<string>()
+const description = ref<string>()
+const domain = ref('wallet.universalprofile.cloud')
```

Comment:
I think this should be `document.location.hostname`. But I think the metadata headers won't work unless it's SSR. Hmmm, what's the use case for this data?

Reply:
I will check with `document.location.hostname`. This is for adding proper `meta` tags to the page. We don't use SSR for this project.

Reply:
I See. I think Twitter metadata will only work with SSR. Need to debug

Reply:
Alright, alternative is to hardcode this things.